packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -67,6 +73,18 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
         );
     }
 
+    const historyViews: React.ReactElement[] = [];
+    for (const h of history.values()) {
+        historyViews.push(
+            <ul key={h.audienceMemberId}> <li>
+                <b>Id: </b>{h.audienceMemberId}
+                <br /><b>Time: </b> {new Date(h.timestamp).toDateString()}
```

Comment:
We should probably print the entire timestamp, and not just the date. We see many ops on the same day, so the date alone probably doesn't give us enough info.

Reply:
On it! Sorry to merge it too soon. I will make another PR.

Reply:
No worries! Sorry, it occurred to me on second look, I missed that you had already merged :)

Reply:
I lost connection on remote desktop, which has everything setup. I'm pulling up from laptop. It might take a while. will let you know when the new pull request is out. Thank you, Josh :)